install/systemd.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: docs
+title: Run Airsonic with Systemd
+permalink: /docs/install/systemd/
+---
+##### Systemd
+
+Systemd is an init system for most linux systems. It allows one to run
+airsonic on boot.
+
+1. Setup dedicated airsonic user
+
+```
+useradd airsonic
+```
+
+2. Setup Airsonic data dir
+
+```
+mkdir /var/airsonic
+chown airsonic /var/airsonic
+```
+
+3. Download war
+
+```
+wget https://github.com/airsonic/airsonic/releases/download/v10.0.0/airsonic-jdbc-extra.war --output-document=/var/airsonic/airsonic.war
```

Comment:
I think we could improve this by using the jekyll system and replace the version with a variable.

`wget {{ site.repo }}/download/v{{ site.stable_version }}/airsonic-jdbc-extra.war`

Again I think we should add straight after the standalone docs a systemd section that enhance the standalone installation .
Not as separate .